react/features/visitors/VisitorsListWebsocketClient.ts

```diff
@@ -124,4 +131,59 @@ export class VisitorsListWebsocketClient extends WebsocketClient {
 
         this.stompClient.activate();
     }
+
+    /**
+     * Unsubscribes from both topic and queue subscriptions.
+     *
+     * @returns {Promise<void>}
+     */
+    override unsubscribe(): Promise<void> {
+        return new Promise(resolve => {
+            try {
+                if (this._topicSubscription) {
+                    this._topicSubscription.unsubscribe();
+                    logger.info('Unsubscribed from visitors list topic');
+                    this._topicSubscription = undefined;
+                }
+
+                if (this._queueSubscription) {
+                    this._queueSubscription.unsubscribe();
+                    logger.info('Unsubscribed from visitors list queue');
+                    this._queueSubscription = undefined;
+                }
+
+                // Also call parent unsubscribe in case there are other subscriptions
+                super.unsubscribe().then(() => {
```

Comment:
I don't think this ever happens. This code should always be no-op! Lets remove this!